source/brailleDisplayDrivers/handyTech.py

```diff
@@ -24,7 +24,25 @@
 import bdDetect
 import time
 import datetime
+from ctypes import windll
+import windowUtils
 from driverHandler import BooleanDriverSetting
+import wx
+
+class InvisibleDriverWindow(windowUtils.CustomWindow):
+	className = u"Handy_Tech_Server"
+	def __init__(self, driver):
+		super(InvisibleDriverWindow, self).__init__(u"Handy Tech Server")
+		self.driver = driver
+	def windowProc(self, hwnd, msg, wParam, lParam):
+		if msg == window_message:
+			if wParam == 100 and lParam == 1:

```

Comment:
Were do 100 and 1 stand for? Could you make these constants? Note that constants are capitalised.